### ISSUE TITLE
Instructions for running software seems to be broken

### ISSUE THREAD

**Issue by flaviostutz** [bancodobrasil/stop-analyzing#4]
I want to run Stop Analyzing, but somenthing appears to be missing from the README instructions on how to run it:

![image](https://user-images.githubusercontent.com/7790172/86046243-43108800-ba23-11ea-9d12-9d507f88fe37.png)

**Comment by tiagostutz** [comment]
You are right... as we are at the early stage of the project we currently don't have the docker images that will be used to run it.
I think we should replace this incomplete docker-compose by some instructions stating that we will currently don't have a functional application.

What do you think? Would like to submit a PR on this?

**Comment by vparine** [comment]
@tiagostutz Can I change the Getting Started section?

**Comment by tiagostutz** [comment]
Hi @vparine !
Sure, go ahead, fork the project, make the changes and submit the PR.!

If you need any assistance, please let me know.

See you in the PR! 😉
